framework-api/framework-api-event-source/src/main/java/uk/gov/justice/services/eventsourcing/source/core/EventSourceConstants.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.services.eventsourcing.source.core;
+
+public abstract class EventSourceConstants {
```

Comment:
Wouldn't this be better as a final class with a private constructor?

Reply:
Ok, done.